### DISCUSSION TITLE
Privilege Escalation in CSI secret store driver containers

### DISCUSSION THREAD

**Discussion by shafanaS** [Azure/secrets-store-csi-driver-provider-azure#719] [Q&A]
Is it necessary to have privilege escalation set to `true` in csi secret store driver containers? 
If the privilege escalation can be set to `false`, how can that be done using helm?

**Comment by aramase** [comment]
Hello 👋🏻   

Yes, `privileged: true` is required for the driver pods. It needs privilege to be perform the mount operation.

**Reply by shafanaS** [reply]
Then what about the config "privilegeEscalation" config. Does it require true or false?
